anyway/parsers/location_extraction.py

```diff
@@ -297,15 +301,56 @@ def extract_location_text(text):
     return text
 
 
+def get_related_waze_accident_alert(db, geo_location, newsflash):
+
+    # determine what distance (in kilometers) to look for waze accidents in, according to the newsflash's resolution
+    distance = resolution_to_distance.get(newsflash.resolution, None)
+    if distance is None:
+        # unknown resolution. skip this optimization
+        return None
+
+    # create the bounding box according to the coordinate we have, and the resolution distance
+    bounding_box_polygon_str = get_bounding_box_polygon(
+        geo_location["lat"], geo_location["lon"], distance
+    )
+
+    # find waze alerts in that bounding box, from the recent time delta - and return the first as the related waze alert
+    matching_alert = (
+        db.session.query(WazeAlert)
+        .filter(WazeAlert.alert_type == "ACCIDENT")
+        .filter(
+            WazeAlert.created_at.between(
+                newsflash.date - timedelta(hours=WAZE_ALERT_NEWSFLASH_DELTA_IN_HOURS),
```

Comment:
Since this constant is only used inside the function, it's should probably be local constant.  This way the occasional reader doesn't have to wonder where else it is used.

Reply:
Also, it can be a `timedelta`, making it slightly more type-safe, and making this expression a single line.